src/main.rs

```diff
@@ -1,3 +1,8 @@
+mod job;
+
+use self::job::Job;
+
 fn main() {
-    println!("Hello, world!");
+    let j = Job::new();
+    println!("{:?}", j);
```

Comment:
Temporary, and we may not even have `Job` public in future iterations of the `job` module. We'll need to make a `core` module that assembles all the other parts together.